desktop/cmp/rest/RestGridModel.js

```diff
@@ -55,6 +55,13 @@ export const deleteAction = {
     actionFn: ({record, gridModel}) => gridModel.restGridModel.confirmDeleteRecord(record)
 };
 
+export const bulkDeleteAction = {
+    text: 'Delete',
+    icon: Icon.delete(),
+    intent: 'danger',
+    recordsRequired: true,
+    actionFn: ({gridModel}) => gridModel.restGridModel.confirmBulkDeleteRecords()
+};
```

Comment:
Note from CR in prev branch: 
Could use this everywhere we want delete...rather than keeping old single del around. Then bulk is driven by multi select enabled or not. Is this too dangerous?

Reply:
I think we should just make the delete action *be* bulk delete.  Simpler. If there is one record in the selection it can use the regular rest deletion endpoint and  otherwise use the bulk deletion endpoint, with

Reply:
if you want a version of the delete action that can *only* be used for single deletion, an application could spread in recordsRequired: 1 to the standard action.  But I think the simplest default is true